src/app.ts

```diff
@@ -77,10 +83,10 @@ function createNodeGenerator(driver?: string): Driver {
       return mysql2;
     }
     case "pg": {
-      return pg;
+      return pg as any;
```

Comment:
Instead of marking this as `any`, add a `execlastidDecl` method to the `pg` and `postgres` drivers that raise an error.

Reply:
thanks, fixed!: [`2398fb0` (#13)](https://github.com/sqlc-dev/sqlc-gen-typescript/pull/13/commits/2398fb0abad6b03621693070e3d0ed316919beac)